lib/alephant/broker/cache/cached_object.rb

```diff
@@ -11,41 +11,39 @@ class CachedObject
         DEFAULT_TTL = 10
 
         def initialize(obj)
-          logger.info(event:   'SettingCachedObject',
+          logger.debug(event:  'SettingCachedObject',
                       content: obj,
                       method:  "#{self.class}#initialize")
-
           @s3_obj = obj
         end
 
         def update(obj)
-          logger.info(event:       'UpdatingCachedObject',
+          logger.debug(event:      'UpdatingCachedObject',
                       old_content: @s3_obj,
                       new_content: obj,
                       method:      "#{self.class}#update")
-
           @s3_obj = obj
         end
 
         def updated
           time = metadata[:'head_Last-Modified']
           Time.parse(time)
         rescue TypeError, ArgumentError => error
-          logger.error(method: "#{self.class}#updated", error: error)
+          logger.error(event: 'updateError', method: "#{self.class}#updated", error: error)
```

Comment:
This isn't trying to update anything so it might be slightly misleading. this `updated` method returns the time at which the item was last updated, based on it's metadata. I think `ParsingTimeLastModifiedError` or `CachedObjectLastModifiedError` might be more specific but they are a bit wordy, so perhaps something shorter